test/tests/functional/pbs_user_reliability.py

```diff
@@ -179,11 +179,15 @@ def test_create_resv_from_array_job(self):
         This test confirms that a reservation cannot be created out of an
         array job.
         """
+        nodes = self.server.status(NODE)
+        ncpus = nodes[0]['resources_available.ncpus']
+        j_range = '1-' + ncpus
```

Comment:
If the available ncpus is 1, In that case the job submission will be "qsub -J 1-1"? I tried this in my local vm and that doesn't seem to work. Please verify once. Rest looks to me.

Reply:
oops. I missed that. Thanks for catching.
I have changed the test case to not look for all subjobs to be running.